data/templates/centos-ks

```diff
@@ -9,7 +9,11 @@ keyboard 'us'
 timezone America/Los_Angeles --isUtc
 firewall --enabled --http --ssh
 selinux --permissive
-bootloader --location=mbr --driveorder=sda --append="crashkernel=auth rhgb"
+<% if (version === "6.5") { %>
+  bootloader --location=mbr --driveorder=<%=driveId%> --append="crashkernel=auth rhgb"
+<% } else { %>
+  bootloader --location=mbr --driveorder=<%=driveId%> --boot-drive=<%=driveId%> --append="crashkernel=auth rhgb"
```

Comment:
Add an explanation for this version judgment, CentOS 6.5 doesn't support the option 'boot-drive', however CentOS 7.0 supports.